modules/battery_manager/jsx/batteryManagerIndex.js

```diff
@@ -254,10 +254,10 @@ class BatteryManagerIndex extends Component {
       this.checkDuplicate(test)
       .then((test) => this.validateTest(test))
       .then((test) => this.postData(
-          this.props.testEndpoint+test.id,
+          this.props.testEndpoint+(test.id || ''),
```

Comment:
This was appearing as `undefined` in the network and was confusing. This resolves that issue.